tools/travis/script.sh

```diff
@@ -16,7 +16,7 @@ tools/build_versions.py
 section_end "List.installed.dependencies"
 
 section "Test"
-pytest $TEST_ARGS skimage
+PY_IGNORE_IMPORTMISMATCH=1 pytest $TEST_ARGS skimage
```

Comment:
Is there a way to avoid needing this? This would warrant the creation of a "test" function again (I am the one who removed it). `skimage.test` or something like that.

Reply:
``py.test`` gets confused by the ``.py`` vs ``.so`` files in the directory... do you have a better solution?

Reply:
cython does not have this issue because ``py.test`` ignores ``.pyx`` files...

Reply:
Not exactly. I get why you decided you kept the same extension but as is, `_brief_loop` still looks like fortran code ;) and would run horribly without compilation.

Maybe it is just time to bring back the `skimage.test` function, though I really like the simplicity of just running `pytest --pyargs skimage` since I don't have to remember each project's own special test functions.